roles/openshift_logging/vars/main.yaml

```diff
@@ -1,6 +1,8 @@
 ---
 openshift_master_config_dir: "{{ openshift.common.config_base }}/master"
 es_node_quorum: "{{openshift_logging_es_cluster_size|int/2 + 1}}"
+es_min_masters_default: "{{ (openshift_logging_es_cluster_size | int / 2 | round(0,'floor') + 1) | int }}"
```

Comment:
@sdodson Are you asking me to change it?  @portante the first arg is to 'what decimal you round to' so this basically says round to the 'ones place' and then use 'floor' for rounding instead of default